packages/functions/src/providers/rewards/assets/UniverseStaderlabsAPYProvider.ts

```diff
@@ -0,0 +1,41 @@
+import { Reward } from '@midas-capital/types';
+import axios from 'axios';
+import { AbstractAssetAPYProvider } from './AbstractAssetAPYProvider';
+
+class UniverseStaderlabsAPYProvider extends AbstractAssetAPYProvider {
+  private data:
+    | {
+        value: number;
+      }
+    | undefined = undefined;
+
+  async init() {
+    this.data = await (await axios.get('https://universe.staderlabs.com/polygon/apy')).data;
```

Comment:
@smartdevcom this actually works for all the chains that stader is in. So we should make this generic, and just pass the chain name. E.g. `https://universe.staderlabs.com/bnb/apy`